### PR TITLE
Refactoring: docstrings in graph.py and figure3.py

### PR DESCRIPTION
**Pull request by angelaxliu**

### CODE REVIEW

**Review by Farnazmdi — APPROVED** [review]
Looks good to me. Thanks!

**Comment by Farnazmdi** [comment]
@aarmey would you like to review this PR?
I believe the changes are writing docstring for graph.py, figure3.py, and graphNorms.py.

File: de/ODE.py [hunk 1]
```diff
@@ -24,9 +21,6 @@ def solver(ps, ts):
 
 
 def ODE(y, _, eps, w, alpha):
-    '''The ODE system:
-    Parameters = eps: Value that bound the saturation effect
-                 w: Interaction between components
-                 alpha: Degradation rate
-    '''
+    """ Returns ODE equation. """
+    # Parameters- eps: bounds the saturation effect, w: interaction between components, alpha: degradation rate
```
Comment (aarmey) [inline de/ODE.py:25]: Arguments should go in the docstring.

**Review by aarmey — CHANGES_REQUESTED** [review]
One small change then this is ready.

**Comment by angelaxliu** [comment]
Just updated those ODE params!

**Pull request merged**